src/App.tsx

```diff
@@ -1,9 +1,21 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const getFiltredQuery = () => (query
```

Comment:
this function getFiltredQuery should only filter your movies by getting a query. Remove `: moviesFromServer`